### ISSUE TITLE
Training and test set

### ISSUE THREAD

**Issue by HelkeBaeyens** [HelkeBaeyens/read#6]
Kimberly: "We need a training set. So it is  pressing we decide if we cover all fields or just one field (or a few) like literature/essays/economical, scientifical,…. Papers / ….
The set exists of sets (how do we make this library EXEL? I can ask Mike): “level, text”"

**Comment by HelkeBaeyens** [comment]
We zouden scientifical (agricultural economics) kunnen nemen, omdat dit economisch wellicht de meeste opportuniteiten biedt. Wij gaan ervan uit dat de kennis van het Engels van iedere wetenschapper hoog genoeg is om wetenschappelijke literatuur in zijn vakdomein te lezen, maar dit is niet het geval (denk maar aan Frans- en Spaanstalige landen). In deze landen wordt vooral het domein van agricultural economics gefinancierd en dan zou onze tool echt ingezet kunnen worden.
Abstracts verzamelen van: https://scholar.google.be/scholar?hl=nl&q=agricultural+economics&btnG=&lr=
???

**Comment by HelkeBaeyens** [comment]
Ik kijk nog na in welk formaat!

**Comment by HelkeBaeyens** [comment]
Mogelijke financiering voor jou: http://www.vliruos.be/nl/projectfinanciering/oproepen/oproepdetail/oproep-reisbeurzen-2017_8250/

**Comment by kimberly-moors** [comment]
Ik heb voorlopig wat literatuurteksten genomen voor het testen van mijn programma.
Tevens heb ik een **exel** aangemaakt. Je kunt deze vinden in het **datagedeelte van mijn vork**. U richting vind ik in ieder geval een met veel potentieel. Laten we zoveel mogelijk teksten hieruit verzamelen maar evengoed een paar teksten van andere domeinen erbij betrekken om te zien hoe accuraat het program op deze zou zijn (bijvoorbeeld de 10 teksten die ik er al in heb geplaatst.

**Comment by HelkeBaeyens** [comment]
Hej Kimberley, voor het machine learning gedeelte zou ik corpora nodig hebben in .txt, .json of .csv.
Plain text (.txt) is het simpelste en voldoet voor ons project. Ik zou dan al corpora verzamelen en een seperator toevoegen (vb ---). Wanneer deze corpora geannoteerd zijn, zou de annotatie op het eerste lijntje kunnen komen. Dus: alle stukken corpora in 1 doc, met een af te spreken seperator en de annotatie voor de tekst. Kan dat werken voor jou?

**Comment by kimberly-moors** [comment]
Voor de teksten is dat zeker geen probleem in txt, deze heb ik namelijk zelf niet nodig. Voor de corpus van de woordenschat heb ik op het moment een csv aangemaakt met een scheiding ; .
Dit bleek het makkelijkste voor mij omdat ik dan in EXEL kon werken. Is het goed als deze in CSV blijft? Ik heb namelijk al zo'n 500 entries gemaakt

**Comment by kimberly-moors** [comment]
Ik kan die teksten namelijk uit gelijk welk doc systeem halen dus dit maar echt niets uit voor mij

**Comment by HelkeBaeyens** [comment]
Hej Kimberly, ik vind in jouw fork geen corpora in .txt, .json of .csv file voor mijn machine learning gedeelte. Waar staat dat ergens (kan zijn dat ik erover kijk)....?

**Comment by kimberly-moors** [comment]
De dictionary csv. is de lijst met alle woorden en hun level. Dat is een csv. file
